backend/app/models/base.py

```
@@ -0,0 +1,16 @@
+from typing import Optional
+
+from sqlmodel import Field, SQLModel
+
+
+class BaseModel(SQLModel):
+    """Base model with common fields."""
+
+    id: Optional[int] = Field(default=None, primary_key=True)
+    created_at: datetime = Field(default_factory=datetime.now)
+    updated_at: Optional[datetime] = Field(
+        default=None, sa_column_kwargs={"onupdate": datetime.now}
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)